resources/kcp/charts/mothership-reconciler/templates/deployment.yaml

```diff
@@ -26,6 +26,23 @@ spec:
         - name: {{ . }}
       {{- end }}
       {{- end }}
+      {{- if .Values.global.mothership_reconciler.auditlog.persistence.enabled }}
+      initContainers:
+      - name: chwon
+        image: busybox
```

Comment:
used image is not versioned and not coming from our registry

Reply:
@moelsayed will you take care of that?
We ususally take such images from our gcr
[example](https://github.com/kyma-project/kyma/blob/main/resources/ory/charts/hydra/templates/tests/test-connection.yaml#L15)